### PR TITLE
[nspr] Set build time based on changelog. Fixes JB#59611

### PR DESCRIPTION
**Pull request by mlehtima**

### CODE REVIEW

File: nspr.spec [hunk 1]
```diff
@@ -70,7 +73,7 @@ popd
                  --enable-optimize="$RPM_OPT_FLAGS" \
                  --disable-debug
 
-%make_build
+make SH_DATE="$BUILD_STRING" SH_NOW="$BUILD_TIME" %{?_smp_mflags}
```
Comment (Thaodan) [inline nspr.spec]: Why did you remove %make_build here? it does accept arguments.

**Review by Thaodan — CHANGES_REQUESTED** [review]
See question.

File: nspr.spec (hunk 1 above)
Comment (mlehtima) [reply nspr.spec]: Changed it back.

**Review by vigejolla — APPROVED** [review]
LGTM

**Review by martyone — APPROVED** [review]

**Review by Thaodan — APPROVED** [review]
LGTM

**Pull request merged**